docs/en/seatunnel-engine/resource-manager.md

```diff
@@ -0,0 +1,81 @@
+---
+
+sidebar_position: 9
```

Comment:
Add this page into https://github.com/apache/seatunnel/blob/476d492165ab2747aa0bb1a5f8a6ed74bfe55f20/docs/sidebars.js#L172

Reply:
done